config.js

```diff
@@ -68,12 +68,24 @@ exports.creds = {
   clockSkew: null,
 };
 
+// The names of the policies you set up in your AD B2C tenant
+// Change to reflect the names you used in your AD B2C tenant
+var policyNames = {
+  signin: 'B2C_1_signin',
+  signup: 'B2C_1_signup',
```

Comment:
### Change Property Name
The property "signup" has to be changed to "signUp". The current name doesn't match the .ejs reference to it.